ols-2.md

```diff
@@ -87,20 +87,21 @@ OLS's second cohort (OLS-2) will be conducted from September 2020 until December
 
 # Role Descriptions
 
-## Mentees
+## Project leads or Mentees
```

Comment:
```suggestion
## Project leads (aka Mentees)
```

Otherwise it sort of sounds like project lead and mentee are two different things